internal/slack/slack.go

```diff
@@ -368,3 +371,27 @@ func (c *Client) NotifyK8SPodErrorEvent(ctx context.Context, event *http.PodErro
 	}
 	return err
 }
+
+func (c *Client) NotifyReleaseManagerError(ctx context.Context, service, environment, branch, namespace, authorEmail string, err error) error {
+	if c.muteOptions.ReleaseManagerError {
+		return nil
+	}
+	userID, err := c.getIdByEmail(ctx, authorEmail)
+	if err != nil {
+		// If user id somehow couldn't be found, post the message to #squad-nasa
+		userID = "#squad-nasa"
+	}
+
+	asUser := slack.MsgOptionAsUser(true)
+	attachments := slack.MsgOptionAttachments(slack.Attachment{
+		Title:      fmt.Sprintf(":boom: Release Manager failed :x:"),
+		Color:      MsgColorRed,
+		Text:       fmt.Sprintf("Failed handling event in release manager for:\nService: %s\nEnvironment: %s\nBranch: %s\nNamespace: %s\nError: %s", service, environment, branch, namespace, err),
```

Comment:
I think we should add something actionable here. This will lead to questions like “what do I do now?”

Can we indicate what the operation was and thus message them something like “your release of x to y failed. Please try again”

Reply:
We could do that, and I was working on it. What stopped me was the many different messages depending on the issue. I see that I also forgot the event type, so this is for all promote, release, etc. Maybe we can do a `generateSlackMessage(evtType, service, environment, branch, namespace string)` that can return some of the better cases and fallback to something more common?